src/middleware/express.ts

```diff
@@ -27,32 +28,18 @@ import * as types from '../types/core';
 
 import {makeChildLogger} from './make-child-logger';
 
-export const APP_LOG_SUFFIX = 'applog';
+export const REQUEST_LOG_SUFFIX = '_reqlog';
 
-export interface MiddlewareOptions extends types.Options {
-  level?: string;
-  levels?: winston.config.AbstractConfigSetLevels;
-}
-
-export async function middleware(options?: MiddlewareOptions) {
-  const defaultOptions = {
-    logName: 'winston_log',
-    level: 'info',
-    levels: winston.config.syslog.levels,
-  };
-  options = Object.assign({}, defaultOptions, options);
+export async function makeMiddleware(
+  logger: winston.Logger,
+  options?: types.Options
+) {
+  options = {logName: 'winston_log', ...options};
 
-  const loggingWinstonApp = new LoggingWinston({
-    ...options,
-    logName: `${options.logName}_${APP_LOG_SUFFIX}`,
-  });
-  const logger = winston.createLogger({
-    level: options.level,
-    levels: options.levels,
-    transports: [loggingWinstonApp],
-  });
+  const transport = new LoggingWinston(options);
```

Comment:
Why is the stackdriver transport added here, rather than caller does that when creating the logger? How would I log to console only in local development?